tests/lib/rules/no-extra-parens.js

```diff
@@ -152,7 +154,6 @@ ruleTester.run("no-extra-parens", rule, {
         "var o = { foo: (function() { return bar(); })() };",
         "o.foo = (function(){ return bar(); })();",
         "(function(){ return bar(); })(), (function(){ return bar(); })()",
-        "((function(){ return bar(); })())",
```

Comment:
I thought this pattern should be reported.
But I will revert if it's a problem. ... (it's a bit troublesome...)